app/views/categories/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<!-- app/views/categories/index.html.erb -->
+
+<div class="categories-container">
+<h2>Categories</h2>
+
+<% if @categories.empty? %>
+<h3>There are no categories yet.</h3>
+<% else %>
+<% @categories.each do |category| %>
+  <div class="category-card">
+  <img src="<%= category.icon %>" />
+    <h2><%= category.name %></h2>
+
+    <!-- calculate total amount spent on this category -->
+    <p>Total Spent: $<%= category.deals.sum(:amount) %></p>
+
+    <!-- This is a link to the category's show page -->
+    <%= link_to 'View', category_path(category) %>
+  </div>
+<% end %>
+<% end %>
+
+
+<!-- link to categories new path -->
+<button class="add-category-button">
+<%= link_to 'Add a new category', new_category_path, style: "color: white;" %>
```

Comment:
- [ ] This button is not yet stick to the bottom of the page as requested by the previous reviewer. Yours probably got to the bottom because you added more items. Please check screenshot below and fix the issue :+1: 
![Screenshot from 2023-11-23 11-39-08](https://github.com/Abdulwhbhussain/Budget-app/assets/63087491/84d882f0-5007-4e52-8119-198f4aed353d)

- This button should equally covered the entire weight. Kindly check original image from the previous reviewer comment above :arrow_up:

Reply:
Brother! I worked on it But Unable to modify it like that.